### ISSUE TITLE
Collapsible: A collapsible group (or region) should have attribute "aria-labelledby"

### ISSUE THREAD

**Issue by leifoolsen** [leifoolsen/mdl-ext#92]
If a collapsible group (or region) does not have an "aria-labelledby" attribute, the component should set the attribute to the corresponding "aria-controls" element.

```html
<div id="button-1" class="mdlext-js-collapsible" role="button" 
  aria-expanded="false" aria-controls="group-1">
  <span>Click to toggle</span>
  <i class="mdlext-aria-expanded-more-less"></i>
</div>
<div class="mdlext-collapsible-group" id="group-1" role="group" 
  aria-labelledby="button-1" hidden>
  <p>Content Region #1 goes here ...</p>
</div
```

**Comment by leifoolsen** [comment]
After some experimenting I've concluded that it's not possible to automate the "aria-labelledby" attribute. It's up to the author to add this attribute where/when it is needed.
